### PR TITLE
(RE-6772) Update License file used by windows

### PR DESCRIPTION
**Pull request by McdonaldSeanp**

### CODE REVIEW

File: resources/windows/wix/license/LICENSE.rtf [hunk 1]
```diff
@@ -1,375 +1,512 @@
...
+deep-merge -  MIT License\
+hocon - Apache 2.0\
+minitar - Ruby License\
+net-ssh - MIT License\
+openssl - Openssl Dual License\
+puppet-ca-bundle - Puppet-ca-bundle license\
+win32utils - Ruby License\
+WiX Toolset - Common Public License 1.0\
+Elevate - MIT License\
+ffi - BSD License\
```
Comment (stahnma) [inline resources/windows/wix/license/LICENSE.rtf]: We need boost, libstdc++ and libgcc in here as well.

Comment (McdonaldSeanp) [inline resources/windows/wix/license/LICENSE.rtf]: what are libstdc++ and libgcc licensed under?

**Comment by geoffnichols** [comment]
Overall, this looks good to me. I'm 👍 .

**Comment by jcoconnor** [comment]
👍 Looks good to me.

**Pull request merged**